foxy/ci-nightly-connext.yaml

```diff
@@ -22,7 +22,7 @@ install_packages:
 - ros-noetic-tf2-msgs
 jenkins_job_label: ci-agent
 jenkins_job_priority: 57
-jenkins_job_schedule: 15 23 1-31/3 * *
+jenkins_job_schedule: 15 23 1-31/2 * *
```

Comment:
So if I understand the Jenkins [documentation](https://www.cloudbees.com/blog/how-to-schedule-a-jenkins-job) about this properly, this will execute on these days:

1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31 (I'm not sure what happens in e.g. February; presumably the last 2 are just not run).

Is my understanding correct?